bin/lxa-iobus-server

```diff
@@ -155,12 +160,24 @@ if args.shell:
 print('starting server on http://{}:{}/'.format(args.host, args.port))
 
 try:
-    kwargs = {}
+    kwargs = {
+        'app': app,
+        'host': args.host,
+        'port': args.port,
+        'handle_signals': False,
+        'print': lambda *args, **kwargs: None,
```

Comment:
This syntax confused me for a bit, but seems ok. :)

Reply:
Same for me :)